templates/layouts/application.html

```diff
@@ -9,12 +9,14 @@
   </head>
 
   <body class="{{ .view }}">
+    {{ include "partials/nav" }}
+
     <main class="app-main">
-      <div class="app-main__content">
+    <div class="app-main__content">
```

Comment:
Missing indentation

Reply:
Fixed in ede28fe6bab73e05d5ee2d0535318a7030402edb